lib/services/finamp_settings_helper.dart

```diff
@@ -554,6 +554,20 @@ class FinampSettingsHelper {
         .put("FinampSettings", finampSettingsTemp);
   }
 
+  static void setAudioFadeOutDuration(Duration audioFadeOutDuration) {
+    FinampSettings finampSettingsTemp = finampSettings;
+    finampSettingsTemp.audioFadeOutDuration = audioFadeOutDuration;
+    Hive.box<FinampSettings>("FinampSettings")
+        .put("FinampSettings", finampSettingsTemp);
+  }
+
+  static void setAudioFadeInDuration(Duration audioFadeOutDuration) {
+    FinampSettings finampSettingsTemp = finampSettings;
+    finampSettingsTemp.audioFadeInDuration = audioFadeOutDuration;
+    Hive.box<FinampSettings>("FinampSettings")
+        .put("FinampSettings", finampSettingsTemp);
+  }
+
```

Comment:
Please also register these new settings in the reset method (in `resetAudioServiceSettings()`)